utilities/transactions/write_unprepared_txn.cc

```diff
@@ -34,12 +34,12 @@ bool WriteUnpreparedTxnReadCallback::IsVisibleFullCheck(SequenceNumber seq) {
   return db_->IsInSnapshot(seq, wup_snapshot_, min_uncommitted_);
 }
 
-SequenceNumber WriteUnpreparedTxnReadCallback::MaxUnpreparedSequenceNumber() {
-  auto unprep_seqs = txn_->GetUnpreparedSequenceNumbers();
```

Comment:
Actually, I seem to remember that the construction of this map was what was causing perf regression (but I forgot to fix it). I wonder if we can get most of the perf benefits, just by using `const auto &`.

Reply:
The regression I see is in WritePrepared. I have not benchmarked WriteUnPrepared.